### ISSUE TITLE
Cannot install with pip

### ISSUE THREAD

**Issue by churnikov** [pyrfume/pyrfume#3]
I'm trying to install your package with pip.

```console
pip version 20.0.2
python version Python 3.7.3
```

During installation I get this error:

```
Collecting pyrfume==0.11
  Using cached pyrfume-0.11.tar.gz (29 kB)
    ERROR: Command errored out with exit status 1:
    ....
    FileNotFoundError: [Errno 2] No such file or directory: './requirements.txt'
```

I'll try to install your package from source for now..

**Comment by rgerkin** [comment]
@churnikov Thanks, I will investigate this now.

**Comment by rgerkin** [comment]
@churnikov I believe I have fixed this (works now in a fresh Conda environment).  It was the same `pip` version 20 backwards incompatibility that has been plaguing other projects as well.

**Comment by rgerkin** [comment]
Fixed in 180ad04325cbdaad75910566b26289cda3bb2a75 and made obsolete by #5.
